apidoc/Global/console/console.yml

```diff
@@ -70,3 +70,26 @@ methods:
       - name: message
         summary: The message(s) to log.
         type: Object
+  - name: time
+    summary: Start a timer to track duration of an operation.
+    description: |
+          Begin a timer that can be used to track the duration of an operation.
+          If not label is passed to the function it will default to "default",
+          if a label already exists then the existing label will not be overwritten
+          and a warning will be logged to the console.
+    parameters:
+      - name: label
+        summary: The label to track the timer by, defaults to "default".
```

Comment:
Isn't there an actual `default` property you can set here?

Reply:
Wasn't aware of that, updated to include default and optional in docs. Making a note to familiarise myself with our doc stuff too :)